src/Views/HelloWorld/Index.cshtml

```diff
@@ -0,0 +1,7 @@
+@{
+    ViewData["Title"] = "Index";
+}
+
+<h2>Index</h2>
```

Comment:
Try sending this title using the ViewData attributes (there should be more info on the tutorial on how to do this)